src/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -167,97 +188,124 @@ public FirstChanceTestException(string message) : base(message)
-            Assert.Equal(AppDomain.CurrentDomain.ApplyPolicy(Assembly.GetEntryAssembly().FullName), Assembly.GetEntryAssembly().FullName);
+            string entryAssembly = Assembly.GetEntryAssembly()?.FullName ?? Assembly.GetExecutingAssembly().FullName;
+            Assert.Equal(AppDomain.CurrentDomain.ApplyPolicy(entryAssembly), entryAssembly);
         }
 
         [Fact]
-        public void CreateDomain()
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
+        public void CreateDomainNonNetfx()
         {
             AssertExtensions.Throws<ArgumentNullException>("friendlyName", () => { AppDomain.CreateDomain(null); });
             Assert.Throws<PlatformNotSupportedException>(() => { AppDomain.CreateDomain("test"); });
         }
 
         [Fact]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.NetFramework)]
+        public void CreateDomainNetfx()
+        {
+            Assert.Throws<ArgumentNullException>(() => { AppDomain.CreateDomain(null); });
+            AppDomain.CreateDomain("test");
+        }
+
+        [Fact]
         public void ExecuteAssemblyByName()
         {
-            string name = "TestApp";
-            var assembly = Assembly.Load(name);
-            Assert.Equal(5,  AppDomain.CurrentDomain.ExecuteAssemblyByName(assembly.FullName));
-            Assert.Equal(10, AppDomain.CurrentDomain.ExecuteAssemblyByName(assembly.FullName, new string[2] {"2", "3"}));
-            Assert.Throws<FormatException>(() => AppDomain.CurrentDomain.ExecuteAssemblyByName(assembly.FullName, new string[1] {"a"}));
-            AssemblyName assemblyName = assembly.GetName();
-            assemblyName.CodeBase = null;
-            Assert.Equal(105, AppDomain.CurrentDomain.ExecuteAssemblyByName(assemblyName, new string[3] {"50", "25", "25"}));
+            RemoteInvoke(() => {
+                string name = "TestApp";
+                var assembly = Assembly.Load(name);
+                Assert.Equal(5, AppDomain.CurrentDomain.ExecuteAssemblyByName(assembly.FullName));
+                Assert.Equal(10, AppDomain.CurrentDomain.ExecuteAssemblyByName(assembly.FullName, new string[2] { "2", "3" }));
+                Assert.Throws<FormatException>(() => AppDomain.CurrentDomain.ExecuteAssemblyByName(assembly.FullName, new string[1] { "a" }));
+                AssemblyName assemblyName = assembly.GetName();
+                assemblyName.CodeBase = null;
+                Assert.Equal(105, AppDomain.CurrentDomain.ExecuteAssemblyByName(assemblyName, new string[3] { "50", "25", "25" }));
+                return SuccessExitCode;
+            }).Dispose();
         }
 
         [Fact]
         [ActiveIssue("https://github.com/dotnet/corefx/issues/18718", TargetFrameworkMonikers.Uap)] // Need to copy files out of execution directory
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Access issues when copying assemblies")]
```

Comment:
it couldn't copy or remove path. I didn't dig into that. I'm already improving current state by enabling those tests and since this is not that much related to UAP I decided to only do quick fixes or leave them as they were